src/EnergyPlus/UnitVentilator.cc

```diff
@@ -3709,301 +3169,224 @@ namespace UnitVentilator {
         if (present(OpMode)) {
             FanOpMode = OpMode;
         } else {
-            FanOpMode = ContFanCycCoil;
+            FanOpMode = DataHVACGlobals::ContFanCycCoil;
```

Comment:
Make this into a C++ default argument. And the next one too.

Reply:
By this do you mean:

    int FanOpMode = DataHVACGlobals::ContFanCycCoil;
    if (present(OpMode)) FanOpMode = OpMode;

Reply:
Or the default argument in the function argument list makes more sense, good idea.